docs/manual/developer_guide.adoc

```diff
@@ -368,53 +368,23 @@ docker cp scap-security-guide:/home/oscap/content/build $(pwd)/container_build
 Under the top level directory, there are directories and/or files for different products,
 shared content, documentation, READMEs, Licenses, build files/configuration, etc.
 
-For example:
-
-[source,bash]
-----
-$ ls -1 scap-security-guide
-build
-build_config.yml.in
-build-scripts
-chromium
-cmake
-CMakeLists.txt
-Contributors.md
-Contributors.xml
-debian8
-DISCLAIMER
-Dockerfile
-docs
-eap6
-fedora
-firefox
-fuse6
-jre
-LICENSE
-linux_os
-ocp3
-ol7
-opensuse
-README.md
-rhel6
-rhel7
-rhosp13
-shared
-sle11
-sle12
-tests
-ubuntu1404
-ubuntu1604
-ubuntu1804
-utils
-wrlinux
-----
-
 ===== Important Top Level Directory Descriptions
 
 |===
 |Directory |Description
 
+|`linux_os`
+| Contains security content for Linux operating systems. Contains rules, OVAL checks, Ansible tasks, Bash remeditions, ..
+
+|`applications`
+| Contains security content for applications such as OpenShift or OpenStack.
```

Comment:
Should `Contains rules, OVAL checks, Ansible tasks, Bash remeditions, ..` be added here as well?

Reply:
fixed